tests/__init__.py

```diff
@@ -0,0 +1,106 @@
+# SPDX-License-Identifier: MIT
+
+import os
+import sys
+f = os.readlink(__file__) if os.path.islink(__file__) else __file__
+path = os.path.realpath(os.path.join(f, "..", "..", "src"))
+
+if path not in sys.path:
+    sys.path.insert(0, path)
+
+import fcntl  # noqa: 402
+import threading  # noqa: 402
+
+import libevdev  # noqa: 402
+import pyudev  # noqa: 402
+
+import ratbag_emu  # noqa: 402
+
+import pytest  # noqa: 402
+import shutil  # noqa: 402
+import subprocess  # noqa: 402
+
+from pathlib import Path  # noqa: 402
+from time import sleep  # noqa: 402
+
+from ratbag_emu.util import EventData  # noqa: 402
+
+
+class TestBase(object):
```

Comment:
shouldn't this class be defined in a separate file?

Reply:
I don't believe so, this isn't an actual test. This a base class for the tests, it defines a bunch of fixtures and some utility functions. The tests classes should subclass it and define their tests.